server/http.go

```diff
@@ -193,6 +193,30 @@ func (p *Plugin) completeUserOAuthToZoom(w http.ResponseWriter, r *http.Request)
 	}
 }
 
+func (p *Plugin) getMattermostUserIdFromZoomParticipant(participant zoom.Participant) (string, error) {
+	conf := p.getConfiguration()
+
+	if !conf.EnableOAuth {
+		return "", errors.New("can't fetch user for JWT zoom app")
+	}
+
+	if conf.AccountLevelApp {
+		user, appErr := p.API.GetUserByEmail(participant.Email)
+		if appErr != nil {
+			return "", errors.Wrap(appErr, "failed to get mattermost user for zoom email")
+		}
+
+		return user.Id, nil
+	}
+
+	info, err := p.fetchOAuthUserInfo(zoomUserByZoomID, participant.ParticipantUserID)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to fetch OAuth user info for zoom user")
```

Comment:
We should try to be consistent with the capitalization of `Zoom`

```suggestion
		return "", errors.Wrap(err, "failed to fetch OAuth user info for Zoom user")
```